src/components/OfflineWithFeedback.tsx

```diff
@@ -107,9 +107,9 @@ function OfflineWithFeedback({
                     return child;
                 }
 
-                const childProps: {children: React.ReactNode | undefined; style: AllStyles} = child.props;
+                const childProps = child.props as {children?: React.ReactNode; style?: AllStyles};
```

Comment:
Could you extract `{children?: React.ReactNode; style?: AllStyles}` to a type? Also using `ChildrenProps`?